### PR TITLE
Versioning

### PR DESCRIPTION
**Pull request by VasavanThiru**

### CODE REVIEW

File: include/xeus-calc/xeus_calc_interpreter.hpp [hunk 1]
```diff
@@ -1,17 +1,17 @@
 #ifndef XEUS_CALC_INTERPRETER_HPP
 #define XEUS_CALC_INTERPRETER_HPP
 
+#include "xeus_calc_config.hpp"
+
 #include "xeus/xinterpreter.hpp"
 
 #include "nlohmann/json.hpp"
```
Comment (JohanMabille) [inline include/xeus-calc/xeus_calc_interpreter.hpp:6]: include dependencies' headers before headers from the library

File: .appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - mkdir build
+  - cd build
+  - cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%MINICONDA%\\Library -D XEXTRA_JUPYTER_DATA_DIR=%MINICONDA%\\share\\jupyter -D BUILD_TESTS=ON ..
+  - nmake
+  - nmake install
+  # Install test dependencies
+
+
+build_script:
+  - .\test\test_xeus_calc
```
Comment (JohanMabille) [inline .appveyor.yml]: This file should be added in a dedicated PR

File: CMakeLists.txt [hunk 3]
```diff
@@ -46,10 +62,17 @@ set(XEUS_CALC_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src)
...
-add_library(xeus-calc SHARED ${XEUS_CALC_SRC})
+set(XEUS_CALC_HEADERS
+    include/xeus-calc/xeus_calc_interpreter.hpp
+    include/xeus-calc/xeus_calc_config.hpp
+)
+
+# xeus-calc is the target for the library
+add_library(xeus-calc SHARED ${XEUS_CALC_SRC} ${XEUS_CALC_HEADERS})
+
+# xeus-calc is the target for the kernel executable
```
Comment (JohanMabille) [inline CMakeLists.txt]: typo

File: test/CMakeLists.txt [hunk 4]
```diff
@@ -4,7 +4,7 @@ if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
     project(xeus_calc-test)
 
     find_package(xeus REQUIRED CONFIG)
-    set(XTENSOR_INCLUDE_DIR ${xtensor_INCLUDE_DIRS})
+    set(XEUS_INCLUDE_DIR ${xeus_INCLUDE_DIRS})
```
Comment (JohanMabille) [inline test/CMakeLists.txt]: You're not building xeus

**Review by JohanMabille — CHANGES_REQUESTED** [review]

**Review by JohanMabille — APPROVED** [review]

**Comment by slel** [comment]
I think "versioning" has no double-n in English.
